source/mapselectmenu.cpp

```diff
@@ -474,13 +486,65 @@ static EntranceNameList * scanScript(const int * script)
                 elvs = reinterpret_cast<spm::machi::ElvDesc *>(script[2]);
                 elvCount = script[3];
             }
+            else if (func == (u32) spm::evt_sub::evt_sub_get_entername)
+            {
+                const int * next_script = script + cmdn + 1;
+                const short * next_p = reinterpret_cast<const short *>(next_script);
+                int next_cmd = next_p[1];
+
+                if (next_cmd == 0xc) // if_str_equal
+                {
+                    if (othersCount < OTHERS_MAX)
```

Comment:
I think we should probably be asserting there's space rather than silently skipping them if not